extensions/arc/src/ui/dashboards/postgres/postgresOverviewPage.ts

```diff
@@ -295,6 +409,13 @@ export class PostgresOverviewPage extends DashboardPage {
 		}
 	}
 
+	private refreshServerNodes(): void {
+		if (this._postgresModel.config) {
+			this.podStatusTable.data = this.getPodStatus().map(p => [p.podName, p.type, p.status]);
```

Comment:
Is there any other information - such as the reason - that could be included here to help the customer figure out why the pod is in that state?

Reply:
A resource health dashboard will be created that provides more information about pod conditions and details.